frontend/src/components/Modal/RiderModal.tsx

```diff
@@ -84,9 +104,21 @@ const RiderModal = ({ existingRider, isRiderWeb }: RiderModalProps) => {
           + Add Student
         </Button>
       ) : (
-        <button className={styles.editRiderButton} onClick={openModal}>
-          <img className={styles.editIcon} alt="edit" src={edit} />
-        </button>
+        <div className={styles.twoButtonDiv}>
+          <button className={styles.editRiderButton} onClick={openModal}>
+            <img className={styles.editIcon} alt="edit" src={edit} />
+          </button>
+          <Switch>
+            <Route path="/riders">
+              <button
+                className={styles.deleteStudentButton}
+                onClick={studentDelete}
+              >
+                <img className={styles.trashIcon} alt="trash" src={trashbig} />
+              </button>
+            </Route>
+          </Switch>
+        </div>
```

Comment:
i wouldnt put this code here. it should go in `UserDetail`, since both employees and students will be able to be removed